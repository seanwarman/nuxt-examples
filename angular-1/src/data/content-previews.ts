import { RoomPlannerImages, SectionImages } from '../assets/images';

export interface ContentPreview {
  label: string;
  primaryImage: string;
  icon: string;
  body: string;
  slug: string;
  firstSubsection?: string;
  learnIcon: string;
  designIcon: string;
}

export const contentPreviews: ContentPreview[] = [
  {
    slug: 'camera-and-audio',
    label: 'Camera & Audio',
    firstSubsection: 'camera',
    primaryImage: SectionImages.cameraAndAudio.contentPreviewHero,
    icon: SectionImages.cameraAndAudio.icons.sectionIcon,
    designIcon: SectionImages.cameraAndAudio.icons.designIcon,
    learnIcon: SectionImages.cameraAndAudio.icons.learnIcon,
    body: `We've tuned and tweaked the hardware. Now, position it correctly to
     unlock the best performance.`,
  },
  {
    slug: 'displays-and-extras',
    label: 'Displays & Extras',
    firstSubsection: 'screen',
    primaryImage: SectionImages.displaysAndExtras.contentPreviewHero,
    icon: SectionImages.displaysAndExtras.icons.sectionIcon,
    designIcon: SectionImages.displaysAndExtras.icons.designIcon,
    learnIcon: SectionImages.displaysAndExtras.icons.learnIcon,
    body: `Correctly sizing displays is a key part of the conference experience.
      Plus, get some hints and tips on using the Series One Board 65 and Desk 27,
      developed in collaboration with Avocor,
      in your space.`,
  },
  {
    slug: 'room-layout',
    label: 'Room layout',
    firstSubsection: 'viewing-area',
    primaryImage: SectionImages.roomLayout.contentPreviewHero,
    icon: SectionImages.roomLayout.icons.sectionIcon,
    designIcon: SectionImages.roomLayout.icons.designIcon,
    learnIcon: SectionImages.roomLayout.icons.learnIcon,
    body: `Following a few spatial guidelines can make a huge difference for remote and in-room participants.`,
  },
  {
    slug: 'furniture-lighting',
    label: 'Furniture and lighting',
    firstSubsection: 'lighting',
    primaryImage: SectionImages.furnitureLighting.contentPreviewHero,
    icon: SectionImages.furnitureLighting.icons.sectionIcon,
    designIcon: SectionImages.furnitureLighting.icons.designIcon,
    learnIcon: SectionImages.furnitureLighting.icons.learnIcon,
    body: `Make your meeting room look and feel inviting with simple decorative and lighting tips. Appropriate lighting helps both local and remote participants have a great experience.`,
  },
  {
    slug: 'acoustics',
    label: 'Acoustics',
    firstSubsection: 'separation',
    primaryImage: SectionImages.acoustics.contentPreviewHero,
    icon: SectionImages.acoustics.icons.sectionIcon,
    designIcon: SectionImages.acoustics.icons.designIcon,
    learnIcon: SectionImages.acoustics.icons.learnIcon,
    body: `Poor acoustics is a common downfall of video conferencing spaces. The right allowance of acoustic finishes can help make speech clear and audible, and adequate isolation helps avoid interruptions.`,
  },
  {
    slug: 'services',
    label: 'Services',
    firstSubsection: 'air-handling',
    primaryImage: SectionImages.services.contentPreviewHero,
    icon: SectionImages.services.icons.sectionIcon,
    designIcon: SectionImages.services.icons.designIcon,
    learnIcon: SectionImages.services.icons.learnIcon,
    body: `Make sure power is provisioned for the right places, and ensure ventilation systems don't cause unwanted noise pickup on your microphones.`,
  },
  {
    slug: 'room-planner',
    label: 'Room Designer',
    primaryImage: RoomPlannerImages.contentPreviewHero,
    icon: RoomPlannerImages.icons.mainIcon,
    designIcon: SectionImages.services.icons.designIcon,
    learnIcon: SectionImages.services.icons.learnIcon,
    body: `Need to size up your room? All the best practices in this guide are used to help you pick the equipment and recommended room size for the occupancy you need.`,
  },
];
