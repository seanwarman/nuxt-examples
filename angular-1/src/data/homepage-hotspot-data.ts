export const HotspotData: Array<{
  name: string;
  text: string;
  color: string;
  defaultIcon: string;
  interimIcon: string;
  activeIcon: string;
  halo: string;
  x: number;
  y: number;
  z: number;
  cameraPosition: {
    x: number;
    y: number;
    z: number;
  };
  cameraRotation: {
    x: number;
    y: number;
    z: number;
  };
  modelsOn: Array<string>;
  modelsOff: Array<string>;
}> = [
  {
    name: 'camera-and-audio',
    text: 'Camera & Audio',
    color: '#e94233',
    defaultIcon: 'icon-hotspot-camera-audio-red-01-default@3x.png',
    interimIcon: 'icon-hotspot-camera-audio-red-01-interim@3x.png',
    activeIcon: 'icon-hotspot-camera-audio-red-01-active@3x.png',
    halo: 'icon-hotspot-camera-audio-red-01-active-halo@3x.png',
    x: -5,
    y: 113,
    z: -160,
    cameraPosition: {
      x: -100,
      y: 175,
      z: 350,
    },
    cameraRotation: {
      x: 100,
      y: 100,
      z: 0,
    },
    modelsOn: ['CameraAndAudio', 'defaultChairs', 'acousticDefault'],
    modelsOff: ['defaultCameraAndAudio'],
  },
  {
    name: 'displays-and-extras',
    text: 'Displays & Extras',
    color: '#0d9cab',
    defaultIcon: 'icon-hotspot-displays-extras-teal-01-default@3x.png',
    interimIcon: 'icon-hotspot-displays-extras-teal-01-interim@3x.png',
    activeIcon: 'icon-hotspot-displays-extras-teal-01-active@3x.png',
    halo: 'icon-hotspot-room-designer-teal-01-active-halo@3x.png',
    x: -230,
    y: 150,
    z: 50,
    cameraPosition: {
      x: 100,
      y: 200,
      z: 500,
    },
    cameraRotation: {
      x: 50,
      y: 100,
      z: 0,
    },
    modelsOn: [
      'DisplaysAndExtras',
      'defaultCameraAndAudioSetup',
      'defaultChairs',
      'defaultCameraAndAudio',
    ],
    modelsOff: [],
  },
  {
    name: 'room-layout',
    text: 'Room Layout',
    color: '#31a750',
    defaultIcon: 'icon-hotspot-room-layout-green-01-default@3x.png',
    interimIcon: 'icon-hotspot-room-layout-green-01-interim@3x.png',
    activeIcon: 'icon-hotspot-room-layout-green-01-active@3x.png',
    halo: 'icon-hotspot-room-layout-green-01-active-halo@3x.png',
    x: 90,
    y: 20,
    z: 200,
    cameraPosition: {
      x: 0,
      y: 280,
      z: 800,
    },
    cameraRotation: {
      x: 150,
      y: 100,
      z: 100,
    },
    modelsOn: [
      'RoomLayout',
      'defaultChairs',
      'defaultCameraAndAudioSetup',
      'acousticDefault',
      'defaultCameraAndAudio',
    ],
    modelsOff: [],
  },
  {
    name: 'furniture-lighting',
    text: 'Furniture & Lighting',
    color: '#fbbc05',
    defaultIcon: 'icon-hotspot-furniture-lighting-yellow-01-default@3x.png',
    interimIcon: 'icon-hotspot-furniture-lighting-yellow-01-interim@3x.png',
    activeIcon: 'icon-hotspot-furniture-lighting-yellow-01-active@3x.png',
    halo: 'icon-hotspot-furniture-lighting-yellow-01-active-halo@3x.png',
    x: 110,
    y: 70,
    z: 80,
    cameraPosition: {
      x: 400,
      y: 200,
      z: 150,
    },
    cameraRotation: {
      x: 0,
      y: 70,
      z: -100,
    },
    modelsOn: [
      'FurnitureAndLighting',
      'defaultCameraAndAudioSetup',
      'acousticDefault',
      'defaultCameraAndAudio',
    ],
    modelsOff: ['defaultChairs'],
  },
  {
    name: 'acoustics',
    text: 'Acoustics',
    color: '#3d85f2',
    defaultIcon: 'icon-hotspot-acoustics-blue-01-default@3x.png',
    interimIcon: 'icon-hotspot-acoustics-blue-01-interim@3x.png',
    activeIcon: 'icon-hotspot-acoustics-blue-01-active@3x.png',
    halo: 'icon-hotspot-acoustics-blue-01-active-halo@3x.png',
    x: -230,
    y: 200,
    z: 180,
    cameraPosition: {
      x: 300,
      y: 800,
      z: -150,
    },
    cameraRotation: {
      x: -150,
      y: 0,
      z: -100,
    },
    modelsOn: [
      'Acoustics',
      'defaultCameraAndAudioSetup',
      'defaultChairs',
      'acousticPanel00',
      'acousticPanel01',
      'acousticPanel02',
      'acousticPanel03',
      'defaultCameraAndAudio',
    ],
    modelsOff: ['stripLight00', 'stripLight01', 'acousticDefault'],
  },
  {
    name: 'services',
    text: 'Services',
    color: '#681da7',
    defaultIcon: 'icon-hotspot-services-purple-01-default@3x.png',
    interimIcon: 'icon-hotspot-services-purple-01-interim@3x.png',
    activeIcon: 'icon-hotspot-services-purple-01-active@3x.png',
    halo: 'icon-hotspot-services-purple-01-active-halo@3x.png',
    x: 110,
    y: 115,
    z: -160,
    cameraPosition: {
      x: 300,
      y: 160,
      z: 150,
    },
    cameraRotation: {
      x: 0,
      y: 50,
      z: -250,
    },
    modelsOn: [
      'services',
      'defaultCameraAndAudioSetup',
      'defaultChairs',
      'acousticDefault',
      'defaultCameraAndAudio',
    ],
    modelsOff: [],
  },
  {
    name: 'room-planner',
    text: 'Room Designer',
    color: '#0d9cab',
    defaultIcon: 'icon-hotspot-displays-extras-teal-01-default@3x.png',
    interimIcon: 'icon-hotspot-displays-extras-teal-01-interim@3x.png',
    activeIcon: 'icon-hotspot-room-designer-teal-01-active@3x.png',
    halo: 'icon-hotspot-room-designer-teal-01-active-halo@3x.png',
    x: -10,
    y: 90,
    z: 0,
    cameraPosition: {
      x: -30,
      y: 150,
      z: 150,
    },
    cameraRotation: {
      x: 40,
      y: 60,
      z: 0,
    },
    modelsOn: [
      'Drawing',
      'defaultCameraAndAudioSetup',
      'defaultChairs',
      'acousticDefault',
      'defaultCameraAndAudio',
    ],
    modelsOff: [],
  },
];
