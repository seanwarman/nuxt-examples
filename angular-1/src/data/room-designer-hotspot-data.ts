export const HotspotData: Array<{
  name: string;
  text: string;
  color: string;
  defaultIcon: string;
  interimIcon: string;
  activeIcon: string;
  halo: string;
  x: number;
  y: number;
  z: number;
  camX: number;
  camY: number;
  camZ: number;
}> = [
  {
    name: 'room-layout',
    text: 'Room Layout',
    color: '#31a750',
    defaultIcon: 'icon-hotspot-room-layout-green-01-default@3x.png',
    interimIcon: 'icon-hotspot-room-layout-green-01-interim@3x.png',
    activeIcon: 'icon-hotspot-room-layout-green-01-active@3x.png',
    halo: 'icon-hotspot-room-layout-green-01-active-halo@3x.png',
    x: -110,
    y: 20,
    z: 100,
    camX: -800,
    camY: 800,
    camZ: 800,
  },
  {
    name: 'camera-and-audio',
    text: 'Camera & Audio',
    color: '#e94233',
    defaultIcon: 'icon-hotspot-camera-audio-red-01-default@3x.png',
    interimIcon: 'icon-hotspot-camera-audio-red-01-interim@3x.png',
    activeIcon: 'icon-hotspot-camera-audio-red-01-active@3x.png',
    halo: 'icon-hotspot-camera-audio-red-01-active-halo@3x.png',
    x: -10,
    y: 90,
    z: 0,
    camX: -800,
    camY: 250,
    camZ: 300,
  },
  {
    name: 'displays-and-extras',
    text: 'Displays & Extras',
    color: '#0d9cab',
    defaultIcon: 'icon-hotspot-displays-extras-teal-01-default@3x.png',
    interimIcon: 'icon-hotspot-displays-extras-teal-01-interim@3x.png',
    activeIcon: 'icon-hotspot-displays-extras-teal-01-active@3x.png',
    halo: 'icon-hotspot-room-designer-teal-01-active-halo@3x.png',
    x: 0,
    y: 110,
    z: -160,
    camX: -1200,
    camY: 500,
    camZ: 600,
  },
  {
    name: 'services',
    text: 'Services',
    color: '#681da7',
    defaultIcon: 'icon-hotspot-services-purple-01-default@3x.png',
    interimIcon: 'icon-hotspot-services-purple-01-interim@3x.png',
    activeIcon: 'icon-hotspot-services-purple-01-active@3x.png',
    halo: 'icon-hotspot-services-purple-01-active-halo@3x.png',
    x: 120,
    y: 200,
    z: -160,
    camX: -120,
    camY: 250,
    camZ: 300,
  },
  {
    name: 'furniture-lighting',
    text: 'Furniture & Lighting',
    color: '#fbbc05',
    defaultIcon: 'icon-hotspot-furniture-lighting-yellow-01-default@3x.png',
    interimIcon: 'icon-hotspot-furniture-lighting-yellow-01-interim@3x.png',
    activeIcon: 'icon-hotspot-furniture-lighting-yellow-01-active@3x.png',
    halo: 'icon-hotspot-furniture-lighting-yellow-01-active-halo@3x.png',
    x: -70,
    y: 265,
    z: 20,
    camX: -120,
    camY: 300,
    camZ: 400,
  },
  {
    name: 'acoustics',
    text: 'Acoustics',
    color: '#3d85f2',
    defaultIcon: 'icon-hotspot-acoustics-blue-01-default@3x.png',
    interimIcon: 'icon-hotspot-acoustics-blue-01-interim@3x.png',
    activeIcon: 'icon-hotspot-acoustics-blue-01-active@3x.png',
    halo: 'icon-hotspot-acoustics-blue-01-active-halo@3x.png',
    x: -210,
    y: 190,
    z: -160,
    camX: -120,
    camY: 220,
    camZ: 580,
  },
  // {
  //   name: 'room-planner',
  //   text: 'Room Planner',
  //   color: '#0d9cab',
  //   defaultIcon: 'icon-hotspot-teal-01@3x.png',
  //   activeIcon: 'icon-room-designer-white-teal-circle-01@3x.png',
  //   halo: 'teal.png',
  //   x: -10,
  //   y: 150,
  //   z: -160,
  //   camX: 0,
  //   camY: 200,
  //   camZ: 200,
  // },
];
