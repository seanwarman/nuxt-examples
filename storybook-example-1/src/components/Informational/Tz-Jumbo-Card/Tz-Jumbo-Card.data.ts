export const exampleJumboCardData = [
  {
    label: {
      enabled: true,
      text: 'New in'
    },
    content: {
      title: 'Best MTB gloves for spring',
      link: {
        url: 'https://google.com',
        target: '_blank',
        internal: false
      }
    },
    image: {
      src: '/Fox-Clothing-Dirtpaw-Long-Finger-MTB-Cycling-Gloves_225074_1_Supersize.jpeg',
      alt: '',
      lifestyle: false
    }
  },
  {
    label: {
      enabled: true,
      text: 'New in'
    },
    content: {
      title: 'Fox Clothing new line 2022',
      link: {
        url: 'https://google.com',
        target: '_blank',
        internal: false
      }
    },
    image: {
      src: '/coen-van-de-broek-xZR-AMfH878-unsplash-3.jpeg',
      alt: '',
      lifestyle: true
    }
  }
];
