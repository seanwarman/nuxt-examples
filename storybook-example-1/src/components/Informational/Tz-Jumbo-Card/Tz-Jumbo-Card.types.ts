import type { LifeStyleImage, Link } from '@/components/Informational/types';

type Label = {
  enabled?: boolean;
  text?: string;
};

type Content = {
  title: string;
  link: Link;
};

export type JumboCards = {
  label: Label;
  content: Content;
  image: LifeStyleImage;
};
