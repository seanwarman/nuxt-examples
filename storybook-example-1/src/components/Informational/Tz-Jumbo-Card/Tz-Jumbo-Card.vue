<template>
  <tz-row :cols="1" :md="2" :gap="5" class="lg:grid-cols-2">
    <tz-col class="tz-jumbo-card" v-for="(card, k) in cards" :key="k">
      <tz-link :link="card.content.link">
        <div
          class="tz-jumbo-card__image-parent mb-[18px] bg-[#F3F3F3]"
          :class="[card.image.lifestyle && 'overflow-hidden']"
        >
          <span
            v-if="card.label.enabled"
            class="tz-jumbo-card__label absolute inset-9 z-20 text-[14px] font-extrabold uppercase tracking-widest"
            >{{ card.label.text }}</span
          >
          <img
            :src="card.image.src"
            :alt="card.image.alt"
            class="absolute top-0 bottom-0 right-0 left-0 m-auto block"
            :class="[card.image.lifestyle ? 'h-full w-full' : 'h-[225px] max-w-full mix-blend-multiply lg:h-[430px]']"
          />
        </div>
        <p class="mb-0 text-[16px] text-[#2F2F2F]">
          {{ card.content.title }}
        </p>
        <!-- supporting product title -->
      </tz-link>
    </tz-col>
  </tz-row>
</template>

<script setup lang="ts">
import { defineProps } from 'vue';
import type { JumboCards } from '@/components/Informational/Tz-Jumbo-Card/Tz-Jumbo-Card.types';

import TzLink from '@/components/Action/Tz-Link/Tz-Link.vue';
import TzCol from '@/components/Structural/Tz-Col/Tz-Col.vue';
import TzRow from '@/components/Structural/Tz-Row/Tz-Row.vue';

export interface Props {
  cards: JumboCards[];
}

defineProps<Props>();
</script>

<style scoped>
.tz-jumbo-card__image-parent {
  @apply relative rounded-[10px];
  padding-top: 134.46%;
}

.tz-jumbo-card:deep(.tz-link img) {
  @apply transition-transform duration-500;
}
.tz-jumbo-card:deep(.tz-link:focus img),
.tz-jumbo-card:deep(.tz-link:hover img) {
  @apply scale-110;
}

.tz-jumbo-card:deep(.tz-link:hover .tz-jumbo-card__image-parent > span) {
  text-shadow: none;
}

.tz-jumbo-card:deep(.tz-link:focus .tz-jumbo-card__image-parent) {
  @apply ring-2 ring-[#2F2F2F] ring-offset-2;
}

.tz-jumbo-card:deep(.tz-link:focus .tz-jumbo-card__label) {
  text-shadow: none;
}
</style>
