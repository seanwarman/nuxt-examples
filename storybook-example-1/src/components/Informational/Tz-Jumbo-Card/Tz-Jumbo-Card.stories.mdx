import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs';

import TzJumboCard from './Tz-Jumbo-Card.vue';
import { exampleJumboCardData } from './Tz-Jumbo-Card.data';

<Meta title="Components/Informational/Jumbo Card" component={TzJumboCard} />

export const Template = (args) => ({
  components: { TzJumboCard },
  setup() {
    return { args };
  },
  template: '<div class="container px-0"><tz-jumbo-card v-bind="args" /></div>'
});

# Jumbo Card

<Canvas withSource="open">
  <Story
    name="Default"
    args={{
      cards: exampleJumboCardData
    }}
  >
    {Template.bind({})}
  </Story>
</Canvas>

<ArgsTable story="Default" />
