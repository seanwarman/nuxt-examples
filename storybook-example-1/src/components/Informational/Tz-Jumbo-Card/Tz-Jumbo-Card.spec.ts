/**
 * @vitest-environment jsdom
 */

import { mount } from '@vue/test-utils';
import TzJumboCard from './Tz-Jumbo-Card.vue';
import { expect, test } from 'vitest';
import { exampleJumboCardData } from './Tz-Jumbo-Card.data';
import type { JumboCards } from '@/components/Informational/Tz-Jumbo-Card/Tz-Jumbo-Card.types';

const wrapper = mount(TzJumboCard, {
  props: { cards: <JumboCards[]>exampleJumboCardData }
});

test('renders correctly', () => {
  expect(wrapper.element).toMatchSnapshot();
});

test('returns a product title', () => {
  expect(wrapper.props().cards[0].content.title).toBeTypeOf('string');
});
