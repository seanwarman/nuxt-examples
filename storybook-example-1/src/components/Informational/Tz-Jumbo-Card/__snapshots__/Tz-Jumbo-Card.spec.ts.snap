// Vitest Snapshot v1

exports[`renders correctly 1`] = `
<div
  class="tz-row grid grid-cols-1 md:grid-cols-2 gap-5 lg:grid-cols-2"
  data-v-5d61d963=""
>
  
  
  <div
    class="tz-col col-span-1 tz-jumbo-card"
    data-v-5d61d963=""
  >
    
    <a
      class="hover:underline focus:underline focus:outline-none"
      data-v-5d61d963=""
      href="https://google.com"
      target="_blank"
    >
      
      <div
        class="tz-jumbo-card__image-parent mb-[18px] bg-[#F3F3F3]"
        data-v-5d61d963=""
      >
        <span
          class="tz-jumbo-card__label absolute inset-9 z-20 text-[14px] font-extrabold uppercase tracking-widest"
          data-v-5d61d963=""
        >
          New in
        </span>
        <img
          alt=""
          class="absolute top-0 bottom-0 right-0 left-0 m-auto block h-[225px] max-w-full mix-blend-multiply lg:h-[430px]"
          data-v-5d61d963=""
          src="/Fox-Clothing-Dirtpaw-Long-Finger-MTB-Cycling-Gloves_225074_1_Supersize.jpeg"
        />
      </div>
      <p
        class="mb-0 text-[16px] text-[#2F2F2F]"
        data-v-5d61d963=""
      >
        Best MTB gloves for spring
      </p>
      <!-- supporting product title -->
      
    </a>
    
  </div>
  <div
    class="tz-col col-span-1 tz-jumbo-card"
    data-v-5d61d963=""
  >
    
    <a
      class="hover:underline focus:underline focus:outline-none"
      data-v-5d61d963=""
      href="https://google.com"
      target="_blank"
    >
      
      <div
        class="tz-jumbo-card__image-parent mb-[18px] bg-[#F3F3F3] overflow-hidden"
        data-v-5d61d963=""
      >
        <span
          class="tz-jumbo-card__label absolute inset-9 z-20 text-[14px] font-extrabold uppercase tracking-widest"
          data-v-5d61d963=""
        >
          New in
        </span>
        <img
          alt=""
          class="absolute top-0 bottom-0 right-0 left-0 m-auto block h-full w-full"
          data-v-5d61d963=""
          src="/coen-van-de-broek-xZR-AMfH878-unsplash-3.jpeg"
        />
      </div>
      <p
        class="mb-0 text-[16px] text-[#2F2F2F]"
        data-v-5d61d963=""
      >
        Fox Clothing new line 2022
      </p>
      <!-- supporting product title -->
      
    </a>
    
  </div>
  
  
</div>
`;
